deploy-service/common/src/main/java/com/pinterest/deployservice/handler/GoalAnalyst.java

```diff
@@ -319,6 +319,11 @@ AgentBean genUpdateBeanByReport(PingReportBean report, AgentBean agent) {
         updateBean.setState(proposeNewAgentState(report, agent));
         updateBean.setStage_start_date(System.currentTimeMillis());
         updateBean.setDeploy_stage(report.getDeployStage());
+        if (report.getExtraInfo() == null) {
+            updateBean.setContainer_Health_Status("N/A (Not Applicable)");
```

Comment:
just empty string to save storage?

Reply:
if there is no such info, we show not applicable instead of empty

Reply:
make sense, updated